tool/gravity/cli/app.go

```diff
@@ -424,7 +424,7 @@ func unpackAppResources(env *localenv.LocalEnvironment, loc loc.Locator, dir, op
 		}
 		defer reader.Close()
 
-		if err := dockerarchive.Untar(reader, dir, archive.DefaultOptions()); err != nil {
+		if err := archive.ExtractGlob(reader, dir, "resources", []string{"*"}); err != nil {
```

Comment:
Also, should the "default" directory (if dir is empty) above be updated to include "resources" directory?

```go
if dir == "" {
	unpackedDir := filepath.Join(env.StateDir, defaults.PackagesDir, defaults.UnpackedDir)
	dir = pack.PackagePath(unpackedDir, loc)
}
```

Reply:
I'd rather have directory argument mandatory.